<?xml version="1.0" ?>
<launch>

  <!-- Start live camera -->
  <node name="live_camera"
  	pkg="usb_cam" type="usb_cam_node" output="screen">
    <rosparam file="$(dirname)/../config/live_camera.yaml"/>
  </node>

  <!-- Start hmi_demo server -->
  <node name="hmi_server"
  	pkg="aist_routines" type="hmi_server.py" output="screen"/>

</launch>
